src/hooks/useSet.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback, useState } from 'react';
+
+type Actions<T> = {
+  has: (key: T) => boolean;
+  add: (key: T) => void;
```

Comment:
The types here need to be updated to take in multiple keys

Reply:
updated